server/src/main/java/com/epam/aidial/core/server/data/ApiKeyData.java

```diff
@@ -23,6 +27,10 @@
  */
 @Data
 public class ApiKeyData {
+    private static final List<String> HTTP_HEADERS_TO_STORE = Stream.of(
+            Proxy.HEADER_JOB_TITLE,
+            Proxy.HEADER_CONVERSATION_ID
+    ).map(String::toLowerCase).toList();
```

Comment:
I don't think we need to make header names to lower case.

You can see GfLogStore tries to get http headers by upper case

Reply:
updated